scripts/gitclone.sh

```diff
@@ -10,23 +10,6 @@ RDIR=user-repo
 READ=gitsource
 FILES=()
 
-# Parse arguments.
```

Comment:
Why is this file `scripts/gitclone.sh` not completely deleted? I don't think this is used outside the chroot sandbox. The gitmanager component does not call any script under the scripts directory, right?

Reply:
Done!